src/config/ConfigCollection.js

```diff
@@ -53,6 +54,27 @@ class ConfigCollection {
     return this.defaultConfigName;
   }
 
+  /**
+   * Set current config version
+   *
+   * @param {string} version
+   * @returns {ConfigCollection}
+   */
+  setConfigVersion(version) {
+    this.configVersion = version;
+
+    return this;
+  }
+
+  /**
+   * Get current config version if set
+   *
+   * @returns {string|null}
+   */
+  getConfigVersion() {
```

Comment:
Config version could be confusing. Like the previous version represents previous changes? ConfigFormatVersion?

Reply:
Using `configFormatVersion`